play-json/jvm/src/test/scala/play/api/libs/json/JsonSpec.scala

```diff
@@ -44,8 +44,8 @@ class JsonSpec extends org.specs2.mutable.Specification {
   implicit val FloatNumbersFormat: Format[FloatNumbers] = Json.format[FloatNumbers]
 
   implicit val PostFormat: Format[Post] = (
-    (__ \ 'body).format[String] and
-    (__ \ 'created_at).formatNullable[Option[Date]](
+    (__ \ Symbol("body")).format[String] and
```

Comment:
Why not use the symbol literal anymore?

Reply:
It's deprecated in Scala 2.13: https://github.com/scala/scala/pull/7395.